Readme.md

```diff
@@ -148,6 +166,59 @@ The current lead maintainer is [Douglas Christopher Wilson](https://github.com/d
 
 [List of all contributors](https://github.com/expressjs/express/graphs/contributors)
 
+### Technical Committee
+
+* [Ulises Gascon](https://github.com/UlisesGascon) -
+  **Ulises Gascon** (he/him)
+* [Jon Church](https://github.com/jonchurch) -
```

Comment:
instead of listing the names twice, what if we listed the github handle as the link, and then the real name (if user has one in their profile) following?

Reply:
```suggestion
* [jonchurch](https://github.com/jonchurch) -
```